app/models/chargeback_configured_system.rb

```diff
@@ -0,0 +1,99 @@
+class ChargebackConfiguredSystem < Chargeback
+  set_columns_hash(
+    :configured_system_id    => :integer,
+    :configured_system_name  => :string,
+    :configured_system_uid   => :string,
+    :configured_system_guid  => :string,
+    :provider_name           => :string,
+    :tenant_name             => :string,
+    :provider_uid            => :string,
+    :cpu_allocated_metric    => :float,
+    :cpu_allocated_cost      => :float,
+    :cpu_cost                => :float,
+    :fixed_compute_1_cost    => :float,
+    :fixed_compute_2_cost    => :float,
+    :memory_allocated_cost   => :float,
+    :memory_allocated_metric => :float,
+    :memory_cost             => :float,
+    :total_cost              => :float
+  )
+
+  def self.build_results_for_report_ChargebackConfiguredSystem(options)
+    @report_user = User.find_by(:userid => options[:userid])
+
+    @configured_systems = nil
+    build_results_for_report_chargeback(options)
+  end
+
+  def self.where_clause(records, options, region)
+    scope = records.where(:resource_type => "ConfiguredSystem")
+    if options[:tag] && (@report_user.nil? || !@report_user.self_service?)
+      scope_with_current_tags = scope.where(:resource => ConfiguredSystem.find_tagged_with(:any => @options[:tag], :ns => '*'))
+      scope.for_tag_names(options[:tag].split("/")[2..-1]).or(scope_with_current_tags)
+    else
+      scope.where(:resource => configured_systems(region))
+    end
+  end
+
+  def self.extra_resources_without_rollups(region)
+    scope = ManageIQ::Providers::IbmTerraform::ConfigurationManager::ConfiguredSystem
```

Comment:
:-1: for this.  We shouldn't be referencing specific provider plugins from core.  If we need the hardware association and virtual_delegates for this to not blow up we should move those to the core ConfiguredSystem model (see https://github.com/ManageIQ/manageiq-providers-ibm_terraform/pull/59/files#r572875849)